### ISSUE TITLE
Combobox is not getting activated through keyboard

### ISSUE THREAD

**Issue by claudiahub** [backstage/backstage#11289]
## Expected Behavior

Combobox UI component should activate through keyboard

## Actual Behavior

1. Navigate to `Home` on the left navigation. 
2. Navigate to the `Type` combobox.
3. Verify whether the Type combobox is getting activated on pressing enter / space / alt+ enter key or not. 

## Context

Ensure all interactive elements can receive focus and are actionable via keyboard. For example, links use the ENTER key while buttons should allow ENTER and SPACE key.

<img width="1342" alt="image" src="https://user-images.githubusercontent.com/54085737/166651164-353359c9-facd-4416-af5b-57def9d974bc.png">

Involved component(s):

- Select from '@backstage/core-components'

User Impact: 
When any UI component are not accessible by using keyboard then user who rely on keyboard are facing difficulty and not able to perform any action and are completely block to perform any functionality on that control element.

A11y Conformance Level: A

WCAG Success Criteria: 2.1.1

## Environment

Browser Information: 
Google Chrome v100.0.4896.127
Firefox 99.0.1
Safari 15.4
